test/test_market_makers.js

```diff
@@ -1,61 +1,57 @@
 const _ = require('lodash')
-const testGas = require('@gnosis.pm/truffle-nice-tools').testGas
+const rlp = require('rlp')
 const { wait } = require('@digix/tempo')(web3)
 
 const utils = require('./utils')
 const { ONE, isClose, lmsrMarginalPrice, getParamFromTxEvent, getBlock, assertRejects, Decimal, randnums } = utils
-const { toBN, soliditySha3, toHex } = web3.utils
+const { toBN, soliditySha3, toHex, keccak256, toChecksumAddress } = web3.utils
 
 const PredictionMarketSystem = artifacts.require('PredictionMarketSystem')
 const LMSRMarketMakerFactory = artifacts.require('LMSRMarketMakerFactory')
 const LMSRMarketMaker = artifacts.require('LMSRMarketMaker')
 const WETH9 = artifacts.require('WETH9')
 
-const contracts = [PredictionMarketSystem, LMSRMarketMakerFactory, LMSRMarketMaker, WETH9]
-
 contract('MarketMaker', function(accounts) {
     let pmSystem
     let lmsrMarketMakerFactory
     let etherToken
-
-    before(testGas.createGasStatCollectorBeforeHook(contracts))
-    after(testGas.createGasStatCollectorAfterHook(contracts))
-
+    let nonce = 0x01
```

Comment:
Oh boy... I wonder if there's a better way to do this.

Reply:
I don't know of one, other than perhaps creating a helper function, for clarity.

Reply:
It's not that big of a deal tbh